### DISCUSSION TITLE
SiteTitle property wont pull into filters webpart

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#3121] [Q&A]
Hi there.. I am sure I am missing something.. But I cannot get this to work for the life of me!
In my screenshot here you can see there is a SiteTitle property if I put my search results into debug view.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/22782498/41b69b4e-8e73-4fd3-a20b-797dc3c5d6e4)
When I try to link the Filters webpart up to the property it wont pull anything in. If does however work for siteURL and other properties so its hooked up as it should be. Any help would be appreciated.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/22782498/3c03eaf8-2e89-40ea-a35e-754929f4dfa8)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/22782498/754bd9c7-acf7-402d-9a19-b866861e9f58)

**Comment by wobba** [comment]
If you check the search schema or https://learn.microsoft.com/en-us/sharepoint/technical-reference/crawled-and-managed-properties-overview you see that SiteTitle is not refinable, thus cannot be used in the filter web part.

If you want this you need to map the same crawled property used in SiteTitle to e.g. RefinableString100, recrawl all sites and use that property instead.

**Reply by nicolesherman00** [reply]
Ahhhh I see... the crawled property is referred to as "sitename" instead of sitetitle like it is everywhere else. Will try to map that one and see if it works. Thank you!
